src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -443,9 +443,10 @@ internal CSharpCompilationOptions WithMemorySafetyRules(int version)
         }
 
         // PROTOTYPE: determine what the "updated" number should be
-        private const int UpdatedMemorySafetyRulesVersion = 2;
+        internal const int UpdatedMemorySafetyRulesVersion = 2;
```

Comment:
Could we use `MemorySafetyRulesAttributeVersion.Updated` instead of an integer value for `WithMemorySafetyRules`?

Reply:
That doesn't feel right to me. That enum is a private implementation detail of PEModuleSymbol (let me actually mark it as `private`) and it is essentially the "interpretation" of the metadata value and has additional values like `Uninitialized` and `NoAttribute` which don't correspond to any real values of the `version` parameter of the MemorySafetyRulesAttribute.

Whereas this constant is supposed to be the real metadata value.